tests/test_examples.py

```diff
@@ -248,6 +249,10 @@ def test_early_stopping(self):
         testargs = ["examples/by_feature/early_stopping.py"]
         run_command(self.launch_args + testargs)
 
+    def test_profiler(self):
```

Comment:
At least on my machine, this takes a significant amount of time to run, should `@slow` be added?

Reply:
Thank you :) I totally agree with your comment. I added a `slow` decorator to the function.